### PR TITLE
Make PR manager use the default branch

### PR DESCRIPTION
**Pull request by Albertoimpl**

### CODE REVIEW

File: scripts/set-pipelines.sh [hunk 1]
```diff
@@ -27,13 +27,15 @@ set_branch_pipeline() {
 
 set_pr_manager_pipeline() {
 	local -r pipeline_name="app-broker-pull-requests${PIPELINE_NAME_SUFFIX:+"-$PIPELINE_NAME_SUFFIX"}"
+  local -r branch="2.2.x"
```
Comment (spikymonkey) [inline scripts/set-pipelines.sh]: Indentation looks off

**Review by spikymonkey — CHANGES_REQUESTED** [review]
Thanks @Albertoimpl , just a small whitespace issue

File: scripts/set-pipelines.sh (hunk 1 above)
Comment (Albertoimpl) [reply scripts/set-pipelines.sh]: oh, thanks! Fixed

**Review by spikymonkey — APPROVED** [review]
Thanks!

**Pull request merged**